doc/bitwise_shift.rst

```diff
@@ -0,0 +1,30 @@
+The BitwiseShift Extension
+===================
+
+The BitwiseShift extensions provides the following operators:
+
+* ``shl``
+* ``shr``
```

Comment:
Textual for consistensy (b-and, b-or etc.)
shl/shr - shift left/right like in asm and pascal for example.